pkg/connector/backfill.go

```diff
@@ -147,7 +156,7 @@ func (tc *TwitterClient) FetchMessages(ctx context.Context, params bridgev2.Fetc
 		Messages: converted,
 		HasMore:  bundle != nil || inbox.Status == types.PaginationStatusHasMore,
 		Forward:  params.Forward,
-		MarkRead: lastReadID == messages[len(messages)-1].ID,
+		MarkRead: shouldMarkRead(lastReadID, messages[len(messages)-1].ID),
```

Comment:
Shouldn't invent new functions when there's already message ID parsing in twittermeow, plus existing sort functions just use string comparisons https://github.com/mautrix/twitter/blob/main/pkg/twittermeow/methods/methods.go

```suggestion
		MarkRead: strings.Compare(lastReadID, messages[len(messages)-1].ID) >= 0,
```

maybe add `lastReadID != "" &&` if necessary

Reply:
Those look like it changed some time in 2018, but that is recent enough that it needs to be supported